lib/ansible/modules/cloud/ovirt/ovirt_hosts.py

```diff
@@ -158,6 +158,11 @@
     state: iscsidiscover
     name: myhost
 
+# login to iscsi targets
+- ovirt_hosts:
+    state: iscsilogin
+    name: myhost
+
```

Comment:
This is an example of a call from a playbook:

                  - ovirt_hosts:
                          state: iscsilogin
                          # TODO: Choose active host in the DC
                          name: "{{ host_name }}"
                          auth: "{{ ovirt_auth }}"
                          iscsi:
                                  username: "{{ item.0.username }}"
                                  password: "{{ item.0.password }}"
                                  address: "{{ item.0.address }}"
                                  target: "{{ item.1 }}"
                    with_subelements:
                          - "{{ import_iSCSI }}"
                          - target

var file includes:
import_iSCSI:
        # TODO: Add support for connect to multiple targets with the same LUN.
        # Every connect should be done using a different ip
        - {id: "e796dbf3-0b60-4087-ac39-b67e05f062ee",
           address: "10.35.16.55",
           target: ["iqn.2015-07.com.mlipchuk2.redhat:444", "iqn.2015-07.com.mlipchuk3.redhat:444"],
           username: "",
           password: "",
           DC_name: "test",
           name: "iscsi"}